src/main/java/exception/DukeException.java

```diff
@@ -0,0 +1,12 @@
+package exception;
+
+/** Planned exceptions thrown by bot. */
+public class DukeException extends Exception {
+
+    /** Constructor helps format error message. */
+    public DukeException(String errorMessage) {
+        super("____________________________________________________________\n"
```

Comment:
perhaps can let Ui.java handle the line formatting to recycle code

Reply:
Thank you very much for all your comments! I've made the changes you suggested- definitely very helpful